doc/whats_new/v1.2.rst

```diff
@@ -26,6 +26,15 @@ Changelog
   boolean. The type is maintained, instead of converting to `float64.`
   :pr:`25147` by :user:`Tim Head <betatim>`.
 
+:mod:`sklearn.metrics`
```

Comment:
We need to move this entry in 1.3 because this is a feature and 1.2.1 will only contain bug fixes